node/config.test.ts

```diff
@@ -21,6 +21,9 @@ const importMapFile = {
   },
 }
 
+const invalidDefaultExportErr = (path: string) =>
+  `Default export in edge function at '${path}' must be a function. More on the Edge Functions API at https://ntl.fyi/edge-api`
```

Comment:
Small nit that you're free to ignore, but I feel that we could make the message shorter by not repeating "edge function".

```suggestion
  `Default export in '${path}' must be a function. More on the Edge Functions API at https://ntl.fyi/edge-api.`
```